lib/base_service.js

```diff
@@ -199,7 +207,7 @@ BaseService.prototype._loadConfig = function _loadConfig(conf) {
         var configFile = this.options.configFile;
         if (/^\./.test(configFile)) {
             // resolve relative paths
-            configFile = path.resolve(self._basePath + '/' + configFile);
+            configFile = path.resolve(process.cwd() + '/' + configFile);
```

Comment:
This is actually more precise, the relative path to the config should be resolved relative to the working directory, which is an expected behaviour.